### PR TITLE
Mobile Website 

### PR DESCRIPTION
**Pull request by AbuRayhaan**
##Portfolio: Mobile setup skeleton

In this milestone, I did the following:
- Fixed the alignment issue by restructuring the index.html file.
- Updated the styles.css file.
- Updated the screenshot image.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<body>
+    <div class="container">
+        <!--Header-->
+        <div class="header">
+            <h1 class="logo">Welcome</h1>
+            <a href=""><img src="icon.png" alt=""></a>
+        </div>
+        <!--Section-->
+        <div class="section">
+            <h1 class="info">I'm Hammed<br>Glad to see you!</h1>
```
Comment (mahbubul14) [inline index.html:19]: - [ ] Currently, your following design does not match with the Figma design. Please work on that. :writing_hand: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
a1 | a2

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @AbuRayhaan ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Pull Request should have a descriptive title telling you what changes are inside and a short summary describing introduced changes in more detail. Please work on that.
p1

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Clean code.

### Optional suggestions

- [ ] Please use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version. It will make your website more responsive.

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized. `Ex: "Fix bug" instead of "fixing bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @AbuRayhaan, 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

## Highlights Highlights Good Points: 👍
- Nice Job so far 👏

## Required Changes ♻️
- [ ] Please your Pull Request should have a descriptive title telling you what changes are inside
Fix it as the previous review said. 

- [ ] CHECK THE COMMENTS UNDER THE REVIEW⬇..

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- Kindly make sure that your Pull Request has a short summary describing introduced changes in more detail


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
Hi @AbuRayhaan, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**